book/doctrine.rst

```diff
@@ -386,9 +391,10 @@ Generating Getters and Setters
 
 Even though Doctrine now knows how to persist a ``Product`` object to the
 database, the class itself isn't really useful yet. Since ``Product`` is just
-a regular PHP class, you need to create getter and setter methods (e.g. ``getName()``,
-``setName()``) in order to access its properties (since the properties are
-``protected``). Fortunately, Doctrine can do this for you by running:
+a regular PHP class with ``protected`` properties, you need to create ``public``
+getter and setter methods (e.g. ``getName()``, ``setName($name)``) in order
+to access its properties. Fortunately, the following command can generate
+these boilerplate methods automatically:
```

Comment:
And actually, maybe we should drop mentioning the command completely. The Doctrine team discourages using them and modern IDEs do generate getter and setter methods for properties quite well.